server/src/main/java/org/opensearch/OpenSearchServerException.java

```diff
@@ -1196,6 +1196,14 @@ public static void registerExceptions() {
                 V_2_7_0
             )
         );
+        registerExceptionHandle(
+            new OpenSearchExceptionHandle(
+                org.opensearch.common.breaker.RequestLevelCircuitBreakingException.class,
+                org.opensearch.common.breaker.RequestLevelCircuitBreakingException::new,
+                171,
+                V_2_7_0
+            )
+        );
```

Comment:
Curious if the generic `CircuitBreakingException` should suffice since circuit breaker is also at a request level today